modules/index/src/main/java/org/apache/ignite/internal/index/IndexBuilder.java

```diff
@@ -272,5 +325,50 @@ private List<RowId> collectRowIdBatch(RowId nextRowIdToBuild) {
 
             return createBatchRowIds(nextRowIdToBuild, BUILD_INDEX_ROW_ID_BATCH_SIZE);
         }
+
+        private boolean enterBusy() {
+            if (!taskBusyLock.enterBusy()) {
```

Comment:
Can't we use one `busyLock` for tasks?

Reply:
Do you think it's right? we want to be able to stop both all tasks and only for a specific index.

Reply:
Should we use busy lock for cases like this? I thought we only use busy lock to stop the whole node